starknet_devnet/postman_wrapper.py

```diff
@@ -106,7 +106,7 @@ async def flush(self, state) -> Tuple[dict, list]:
         """Handles all pending L1 <> L2 messages and sends them to the other layer."""
 
         if self.__postman_wrapper is None:
-            return {}
+            return ([], [])
```

Comment:
Can be `return [], []` but you can leave it as it is if you think it's clearer that way.

Reply:
I'll leave it as it is